src/Mvc/samples/MvcSandbox/MvcSandbox.csproj

```diff
@@ -11,6 +11,7 @@
     <Reference Include="Microsoft.AspNetCore.StaticFiles" />
     <Reference Include="Microsoft.Extensions.Logging.Console" />
     <Reference Include="Microsoft.Extensions.Logging.Debug" />
+    <Reference Include="Microsoft.AspNetCore.ResponseCaching" />
```

Comment:
This addition and the src/Mvc/samples/MvcSandbox/Pages/PagesHome.cshtml changes are odd for a PR about output caching. Maybe they should be handled in a separate PR❔